examples/tomo/checks/check_axes_cone2d_bp.py

```diff
@@ -0,0 +1,64 @@
+"""Cone beam 2D example for checking that orientations are handled correctly.
```

Comment:
This whole file is hard to read. I'd say it requires a notable cleanup.

I'll shortly write here:

* The introduction should make it clear that this is perhaps not meant as a usage example. We could maybe add a readme to `examples/tomo/checks` to further clarify this.
* The workaround of first creating a 3d phantom and then a 2d phantom is weird. I'd recommend simply adding a new phantom/2d version of the old phantom
* `opening_angle` is called `fan_angle` in the literature
* The text talks about different backends but the example uses only one. Preferably add a for-loop over backends and if not let users pick in a clear way.
* Overall code complexity is too high, try to reduce